tools/python/get_submodules.py

```diff
@@ -2,14 +2,21 @@
 from pathlib import Path
 import argparse
 import configparser
+import itertools
 import json
 import re
+import urllib.parse
 
 import pygit2
 
+def absolute_url(url):
+    # Support relative URLs (see https://github.com/pybind/pybind11/issues/1831)
+    if url.startswith('../'):
+        return urllib.parse.urljoin('https://github.com/microsoft/onnxruntime', url)
+    return url
 
 def format_component(submod):
-    return {"component": {"type": "git", "git": {"commitHash": str(submod.head_id), "repositoryUrl": submod.url}}}
+    return {"component": {"type": "git", "git": {"commitHash": str(submod.head_id), "repositoryUrl": absolute_url(submod.url)}}}
 
```

Comment:
This formatting order seems to match the original cgmanifest.json and not the updated one in this PR where the 'git' entry is prior to the 'type' entry.

Reply:
Python dicts are hashed dictionaries, with no implicit ordering. The order in which the json is dumped depends on the parameters given to `json.dump` (see line 79 of the script: `json.dump(registrations, f, indent=4, sort_keys=True)`. sort_keys will sort them alphabetically).

Reply:
If someone can point me to how the old one was generated or let me know if it was manually generated please do :)